sdk/keyvault/azure-keyvault-certificates/tests/ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I strongly suspect that credscan is going to flag this (it looks for secrets *accidentally* checked in to repos, but I'm not sure what the right way of excluding "fake" secrets *intentionally* checked in)

Reply:
thankfully not a problem